README.md

```diff
@@ -7,8 +7,9 @@ Whenever there's a change in the Eth app binding or the wrapper, `bundle.js` has
 ### How to generate `bundle.js`
 
 1. Clone the [monorepo](https://github.com/ledgerhq/ledger-live) and compile the libraries using `pnpm build:libs`
-2. Use [browserify](https://browserify.org/) to wrap everything needed for `JavaScriptCore` to run the binding and put it into the `JavaScript` folder inside the Package (should replace the current `bundle.js`) using the following command:
+2. Run the following commands in `Sources/EthWrapper/JavaScript`:
 
 ```
-browserify <path_to_monorepo>/ledger-live/libs/ledgerjs/iOS-wrappers/ios-wrapper-eth/lib/Wrapper.js -o "<path_to_package>/ios-ble-wrapper-eth/Sources/EthWrapper/JavaScript/bundle.js" -d -s TransportModule
+pnpm i
+pnpm build --entry "<path_to_monorepo>/ledger-live/libs/ledgerjs/packages/swift-bridge-hw-app-eth/lib/Wrapper.js"
```

Comment:
What do you use as `<path_to_monorepo>` here? I keep getting node path resolution errors so might need a bit more context on where this package should live.

My errors are eg
<path_to_monorepo>

Reply:
No, just clone the monorepo and this repo into their own separate folders as you would any other repo, and use the path to the ledger-live repo as `<path_to_monorepo>`.

Reply:
> What do you use as `<path_to_monorepo>` here? I keep getting node path resolution errors, so might need a bit more context on where this package should live. git clone this repo into`ledger-live/libs/ledgerjs/packages/`?

Let's say the `ios-ble-wrapper-eth` is cloned in `~/code/ios-ble-wrapper-eth` and the LL monorepo is cloned in `~/code/ledger-live`, then:
- the 2 commands above (`pnpm i` and `pnpm build ...`) should be run in `~/code/ios-ble-wrapper-eth/Sources/EthWrapper/JavaScript`
- the `--entry` arg of the `pnpm build` command should be `~/code/ledger-live/libs/ledgerjs/packages/swift-bridge-hw-app-eth/lib/Wrapper.js` (i.e: the location of the `ledger-live/libs/ledgerjs/packages/swift-bridge-hw-app-eth/lib/Wrapper.js` on your system

Basically, what all this means in english is: "build this `ios-ble-wrapper-eth` project taking the `Wrapper.js` file (located in LL monorepo) as entry point"
cf. [webpack entry doc](https://webpack.js.org/api/cli/#entry) for more context